Doc/c-api/arg.rst

```diff
@@ -518,9 +518,9 @@ Building values
    When memory buffers are passed as parameters to supply data to build objects, as
    for the ``s`` and ``s#`` formats, the required data is copied.  Buffers provided
    by the caller are never referenced by the objects created by
-   :c:func:`Py_BuildValue`.  In other words, if your code invokes :c:func:`malloc`
+   :c:func:`Py_BuildValue`.  In other words, if your code invokes :c:func:`!malloc`
```

Comment:
This is not needed; `malloc` is in the nitignore list in Doc/conf.py.

```suggestion
   :c:func:`Py_BuildValue`.  In other words, if your code invokes :c:func:`malloc`
```

Reply:
I saw warnings on this function on this documentation page. Maybe the PR was outdated, I don't recall the details.

I didn't know nitignore, that sounds very useful!